minecraft_servers/crystopia/manifest.json

```diff
@@ -2,6 +2,9 @@
     "server_name": "crystopia",
     "nice_name": "Crystopia",
     "direct_ip": "crystopia.net",
+    "server_wildcards": [
+    "crystopia.*"
```

Comment:
I think you want this:
```suggestion
    "%.crystopia.de"
```

But that doesn't look like a real wildcard on that domain:
![image](https://github.com/LabyMod/server-media/assets/37186532/2953739f-5ed4-4450-b002-69b66963d267)

Reply:
Yes and no Crystopia.de has currently no Subdomain for the Minecraft Server

Reply:
He doesn't care

Reply:
Well you're right but I just want to add Crystopia.de but if that fits with %.crystopia.de and then Crystopia.de is inside because of background and so on in mc.. then it fits

Reply:
AFAIK you can only add real wildcard domains - that is, only domains that have literally every possible subdomain pointing to your server - to the `server_wildcards` array.

Reply:
Yes but how i can add a other domain...